### PR TITLE
feat: new about us page

### PR DESCRIPTION
**Pull request by jfcurran**
The goal of this pr is to
- add members of core team to the about us page
- remove outdated descriptions and/or refresh them

### CODE REVIEW

File: src/components/layouts/team/team-members.tsx [hunk 1]
```diff
@@ -333,12 +342,5 @@ export const teamCommunity = {
...
   ),
-  img: (
-    <img
-      loading="lazy"
-      alt="The Ory Community"
-      className="responsive"
-      src="https://opencollective.com/ory/contributors.svg?avatarHeight=24&width=540&button=false"
-    />
-  )
+  img: <Contributors />
```
Comment (jfcurran) [inline src/components/layouts/team/team-members.tsx:345]: doing it like this also doesnt work, Ill import it as a static image

**Review by aeneasr — CHANGES_REQUESTED** [review]
Please `authorisation` -> `authorization`

https://english.stackexchange.com/questions/282621/authorization-vs-authorisation-im-in-some-real-dilemma

**Review by aeneasr — CHANGES_REQUESTED** [review]
Everything looks good but the collaborators image!

**Comment by jfcurran** [comment]
@aeneasr  this was due to some hassle - this is the error message, when attempting to load the image using the regular html tag img

```
Refused to load the image 'https://opencollective.com/ory/contributors.svg?avatarHeight=24&width=540&button=false' because it violates the following Content Security Policy directive: "default-src 'self' blob: 'unsafe-inline' 'unsafe-eval' unpkg.com *.algolianet.com *.algolianet.net storage.googleapis.com github.com *.githubusercontent.com gh-card.dev *.ory.sh *.youtube.com *.youtube-nocookie.com fonts.googleapis.com fonts.gstatic.com data: s.ytimg.com *.cloudfront.net". Note that 'img-src' was not explicitly set, so 'default-src' is used as a fallback.
```

to circumwent this for realease, i went ahead and added the image to src and imported it with StaticImage

**Comment by aeneasr** [comment]
Update

https://github.com/ory/web/blob/dbb7051c79e9604af7056dd02b2b165867ea7738/static/_headers#L4

to

```
Content-Security-Policy: default-src 'self' blob: 'unsafe-inline' 'unsafe-eval' unpkg.com *.algolianet.com *.algolianet.net storage.googleapis.com github.com *.githubusercontent.com gh-card.dev *.ory.sh *.youtube.com *.youtube-nocookie.com fonts.googleapis.com fonts.gstatic.com data: s.ytimg.com *.cloudfront.net; img-src * data:,
```

**Review by aeneasr — CHANGES_REQUESTED** [review]
* [ ] File `src/images/community/contributors.svg` can be removed now

**Pull request merged**